content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -194,6 +194,77 @@ For example, if there are 1,253 pods on the cluster and the client wants to rece
 
 Note that the `resourceVersion` of the list remains constant across each request, indicating the server is showing us a consistent snapshot of the pods. Pods that are created, updated, or deleted after version `10245` would not be shown unless the user makes a list request without the `continue` token.  This allows clients to break large requests into smaller chunks and then perform a watch operation on the full set without missing any updates.
 
+## Lists
+
+There are dozens of list types (such as `PodList`, `ServiceList`, and `NodeList`) defined in the Kubernetes API.
+You can get more information about each list type from the [Kubernetes API](https://kubernetes.io/docs/reference/kubernetes-api/) documentation.
+
+When you query the API for a particular type, all items returned by that query are of that type. For example, when you
+ask for a list of services, the list type is shown as `kind: ServiceList` and each `item` in that list represents a service. For example:
+
+```console
+
+GET /api/v1/namespaces/services
+---
+{
+  "kind": "ServiceList",
+  "apiVersion": "v1",
+  "metadata": {
+    "resourceVersion": "2947301"
+  },
+  "items": [
+    {
+      "metadata": {
+        "name": "kubernetes",
+        "namespace": "default",
+...
+      "metadata": {
+        "name": "kube-dns",
+        "namespace": "kube-system",
+...
+```
+
+Some tools, such as `kubectl` provide another way to query the Kubernetes API. Because the output of `kubectl` might include multiple list types, the list of items is represented as `kind: List`. For example:
+
+
+```console
+
+$ kubectl get services -A -o yaml 
+
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: "2021-06-03T14:54:12Z"
+    labels:
+      component: apiserver
+      provider: kubernetes
+    name: kubernetes
+    namespace: default
+...
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      prometheus.io/port: "9153"
+      prometheus.io/scrape: "true"
+    creationTimestamp: "2021-06-03T14:54:14Z"
+    labels:
+      k8s-app: kube-dns
+      kubernetes.io/cluster-service: "true"
+      kubernetes.io/name: CoreDNS
+    name: kube-dns
+    namespace: kube-system
+...
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+```
+
+Keep in mind that the Kubernetes API does not have a `kind: List` type. The `kind: List` resource is more of an internal mechanism for tools such as `kubectl` to present lists of mixed resources in the same output.
```

Comment:
I'd use a `{{< note >}}` callout for this. Also be consistent about whether it's a kind or a type or a resource.

Reply:
I'll make it an note.  As for what to call `kind: List`, I just said it wasn't a "type". And it's certainly not a Kubernetes resource. So I'll drop "The" and "resource" and just start the sentence with `kind: List`, as long as that doesn't break some docs style issue.

Reply:
Done.